articles/142_idl.md

```diff
@@ -35,3 +35,45 @@ Further it describes how the interfaces are used to generate code in C, C++ and
 <div class="alert alert-warning" markdown="1">
   <b>TODO:</b> All of this too
 </div>
+
+### Type Mapping
+
+IDL types mapped to different types in different languages.
+
+
+| IDL type | C type    | C++ type    | Python type          |
+| -------- | --------- | ----------- | -------------------- |
+| bool     | \_Bool    | bool        | bool                 |
+| octet    | uint8\_t  | uint8\_t    | bytes with length 1  |
```

Comment:
`bytes` feels odd since it is a sequence type in python. Thoughts on `ctypes.c_char` or `int` to match `uint8_t` instead?